packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -206,18 +207,24 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
       const kibanaResponse = await handler(kibanaRequest, kibanaResponseFactory);
       return hapiResponseAdapter.handle(kibanaResponse);
     } catch (e) {
+      // log and capture error
       this.log.error(e);
+      apm.captureError(e);
```

Comment:
Perfect! Can you remove the manual error capturing from APM then?

https://github.com/elastic/kibana/blob/995d8f1e01f1c1ae5635e71a044eaa3f8f248de7/x-pack/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts#L188C1-L189

Reply:
I could, but then the error would not be logged, given you're not throwing but returning a `custom` response instead:

https://github.com/elastic/kibana/blob/ac2fc4c3beca7b677910815566db55b93d8406a7/x-pack/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts#L188-L191

Are you sure it would be fine?

Reply:
Oh, interesting. I didn't think of that. In that case, just leave it as-is.